puppet/zulip/files/logrotate/zulip

```diff
@@ -1,8 +1,13 @@
-/var/log/zulip/server.log /var/log/zulip/workers.log /var/log/zulip/manage.log {
+/var/log/zulip/*.log {
 	missingok
 	rotate 10
 	size 1G
 	compress
 	delaycompress
 	notifempty
+        create 644 zulip zulip
```

Comment:
So, I'm a bit puzzled by this line; logrotate seems to do the right thing already in terms of having files it rotate keep the ownership state they had before, but this seems like a clear improvement.  Did you encounter an actual problem here, or is this just making the code more robust?